modules/openxBidAdapter.js

```diff
@@ -174,14 +161,14 @@ function isBidRequestValid(bidRequest) {
   return !!(bidRequest.params.unit && hasDelDomainOrPlatform);
 }
 
-function buildRequests(bids, bidderRequest) {
-  let videoBids = bids.filter(bid => isVideoBid(bid));
-  let bannerAndNativeBids = bids.filter(bid => isBannerBid(bid) || isNativeBid(bid))
+function buildRequests(bidRequests, bidderRequest) {
```

Comment:
NOTE: there is a room for an improvement to remove this custom logic at all, since we support multi-imp multi-adplacement bidrequests(https://github.com/openx/exchange-diagrams/blob/master/misc/multi-glossary.md#multi-imp-multi-ad-placement-bidrequest).